mapie/multi_label_classification.py

```diff
@@ -403,38 +469,6 @@ def _transform_pred_proba(
 
         return np.expand_dims(y_pred_proba_array, axis=2)
 
-    def _compute_risks(self, y_pred_proba: NDArray, y: NDArray) -> NDArray:
-        """
-        Compute the risks
-
-        Parameters
-        ----------
-        y_pred_proba : NDArray of shape (n_samples, n_labels)
-            Predicted probabilities for each label and each observation
-        y : NDArray of shape (n_samples, n_labels)
-            True labels.
-
-        Returns
-        -------
-        NDArray of shape (n_samples, n_lambdas)
-            Risks for each observation and each value of lambda.
-        """
-        y_pred_proba_repeat = np.repeat(
-            y_pred_proba,
-            self.n_lambdas,
-            axis=2
-        )
-
-        y_pred_th = (y_pred_proba_repeat > self.lambdas).astype(int)
-
-        y_repeat = np.repeat(y[..., np.newaxis], self.n_lambdas, axis=2)
-        risks = 1 - (
-            (y_pred_th * y_repeat).sum(axis=1) /
-            y.sum(axis=1)[:, np.newaxis]
-        )
-
-        return risks
-
     def _get_r_hat_plus(
```

Comment:
to be moved in the contrl_risk folder to be consistent with your ltt implementation

Reply:
Done, if you have a better idea for the filename tell me!

Reply:
just not in capital letters, it's kind of agressive ^^